osidb/helpers.py

```diff
@@ -56,3 +58,23 @@ def get_unique_meta_attr_keys(model: Type[models.Model]) -> List[str]:
 def get_model_fields(model: Type[models.Model]) -> List[str]:
     """Get all field names of the model"""
     return [field.name for field in model._meta.get_fields()]
+
+
+class TaskFormatter(logging.Formatter):
+    """
+    Custom formatter based on celery 'celery.utils.log.TaskFormatter'
```

Comment:
can't we use `celery.utils.log.TaskFormatter` directly in the settings?

Reply:
We can, but the base `celery.utils.log.TaskFormatter` injects `???` in log massages where `task_name` or `task_id` cannot be retrieved, eg. the code is executed outside the Celery task, it felt to me kinda disturbing to have bunch of question marks  in the logs and also it allows us to `task_name` which can be switched just for the name when it is missing (again when the code is executed outside of the celery task).

I am not strongly for or against this, so we can use the basic `celery.utils.log.TaskFormatter`, wdyt?

Reply:
in that case I have no opinion either, but maybe it should inherit from the celery TaskFormatter instead of the generic one? not sure if the celery one does extra stuff

Reply:
The celery `TaskFormatter` is defined kinda weird, it inherits from some celery `ColorFormatter` but it overrides its coloring behavior and thus it basically inherits from the base `logging.Formatter` so I would say, that we don't need to inherit from the celery one. Also it is not a `celery.utils.log.TaskFormatter` but `celery.app.log.TaskFormatter`, just for the record.